kubectl-plugin/waiops-status.md

```diff
@@ -127,6 +126,12 @@ Cluster   katamari    aiops-installation-edb-postgres   Cluster in healthy state
 Cluster   katamari    common-service-db                 Cluster in healthy state
 Cluster   katamari    zen-metastore-edb                 Cluster in healthy state
 
+______________________________________________________________
+FlinkDeployment:
+
+NAME                       JOB STATUS   LIFECYCLE STATE
+aiops-ir-lifecycle-flink                STABLE
```

Comment:
We may need to re-run this now that LAD Flink is enabled

Reply:
Completed in [a09a445](https://github.com/IBM/cp4waiops-samples/pull/257/commits/a09a445e0e17975a10732d2fe5509a4ade8da95d)